src/pmpo_createTestMPMesh.cpp

```diff
@@ -56,17 +56,54 @@ Mesh* createMesh(const mesh_type meshType, const geom_type geomType,
 
 Mesh* initTestMesh(const int testMeshOption, const int replicateFactor){
     //add a switch statement testMeshOption
-    if(testMeshOption == 1){ //this is a 2d mesh with 10 tri-oct elements
+   if(testMeshOption==1 || testMeshOption==2){//this is a 2d mesh with 10 tri-oct elements
         const mesh_type meshType = mesh_general_polygonal;
         const geom_type geomType = geom_planar_surf;
         const double sphereRadius = 0.0;
         const int nVertices_size = 19;
         const int nCells_size = 10;
-        const std::vector<std::vector<double>> v_array = //[nVertices_size][vec3d_nEntries]
-            {{0.00,0.00,1.1},{0.47,0.00,1.1},{1.00,0.00,1.1},{0.60,0.25,1.1},{0.60,0.40,1.1}, //5
-             {0.00,0.50,1.1},{0.31,0.60,1.1},{0.40,0.60,1.1},{0.45,0.55,1.1},{0.70,0.49,1.1}, //10
-             {0.80,0.45,1.1},{0.90,0.47,1.1},{1.00,0.55,1.1},{0.60,0.60,1.1},{0.37,0.80,1.1}, //15
-             {0.00,1.00,1.1},{0.37,1.00,1.1},{0.48,1.00,1.1},{1.00,1.00,1.1}};                //19
+        std::vector<std::vector<double>> v_array_coords = //[nVertices_size][vec3d_nEntries]
```

Comment:
I see that `vector<vector<double>>` was used elsewhere in the code, but we should not use it for fixed size arrays (i.e., the size is known at compile time).  Instead, please use something like std::array.  Here is a simple example: https://godbolt.org/z/71MhKeKr6

Reply:
Jun, You do NOT need to create a new PR (to account for changes based on Cameron's comments). Do you know how to do that?

Reply:
Do I push the new changes to this repository?

> Jun, You do NOT need to create a new PR (to account for changes based on Cameron's comments). Do you know how to do that?

Reply:
@Acewvrs If you make the changes in your branch and push them to github the PR will 'automatically' be updated.

Reply:
@cwsmith If I change all std::vector<std::vector<double>> to std::array, wouldn't that cause an error because createMesh() is expecting std::vector<std::vector<double>>? Or should I change the implementation of createMesh() so it takes std::array<> as an argument?

Reply:
@Acewvrs Yeah.  Sorry about that.  I missed that some of these `std::vectors` were being passed into `createMesh(...)`.  Those `std::vectors` can remain 'as is'.

We could create `std::vectors` from the `std::arrays`... but I'm not sure if that is a good idea here, especially since they are nested/2D.